server/protocols/jmap-rfc-8621/doc/specs/spec/mail/message.mdown

```diff
@@ -750,15 +810,27 @@ This would sort Emails in flagged Threads first (the Thread is considered flagge
 
 When *collapseThreads* is `true`, then after filtering and sorting the Email list, the list is further winnowed by removing any Emails for a Thread id that has already been seen (when passing through the list sequentially). A Thread will therefore only appear **once** in the result, at the position of the first Email in the list that belongs to the Thread (given the current sort/filter).
 
+<aside class="warning">
+We are not supported yet
```

Comment:
Body truncation implementation do not take html structure into account.

Truncation can happen in the middle of an html tag.